languages/go/go.mod

```diff
@@ -5,16 +5,10 @@ go 1.14
 require (
 	github.com/fatih/color v1.13.0 // indirect
 	github.com/goccy/go-yaml v1.9.4
-	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/go-cmp v0.5.6
 	github.com/mattn/go-colorable v0.1.11 // indirect
 	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519 // indirect
-	golang.org/x/mod v0.5.1 // indirect
-	golang.org/x/net v0.0.0-20211020060615-d418f374d309 // indirect
 	golang.org/x/sys v0.0.0-20211020174200-9d6173849985 // indirect
-	golang.org/x/text v0.3.7 // indirect
-	golang.org/x/tools v0.1.7 // indirect
-	google.golang.org/appengine v1.6.7 // indirect
-	gorm.io/driver/sqlite v1.2.6 // indirect
-	gorm.io/gorm v1.22.5 // indirect
+	gorm.io/driver/sqlite v1.2.6
```

Comment:
So will installing Oso now require installing sqlite? I have no idea how Go dependencies work

Reply:
https://stackoverflow.com/a/66178556

Reply:
Believe it no longer will now that I've separated out test dependencies into their own `go.mod`. I used gorm as an example: https://github.com/go-gorm/gorm/blob/e5894ca44951fecc3b3f31f1aa46df7de6024b04/tests/go.mod